src/components/text-editor/prosemirror-adapter/plugins/image/inserter.ts

```
@@ -261,19 +283,253 @@
+                `external-image-${Date.now()}.${extension}`;
+            const file = new File([blob], fileName, { type: fileType });
+
+            const reader = new FileReader();
+            reader.onloadend = () => {
+                dispatchImagePastedEvent(view, reader.result as string, file);
+            };
+
+            reader.readAsDataURL(blob);
+        })
+        .catch((error) => {
+            console.error('Error processing external image:', error);
```

ESLint

[error] Unexpected console statement.

(no-console)